object-manipulation-2/src/omit.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-redeclare */
+/* exported omit */
+/*
+variable: unbanned empty array, list out source keys,
+  for each source key, check if it's banned with each item of keys
+  , if not, then add to unbanned array
+once that's done, iterate through unbanned array and add property to obj output
+*/
+
+function omit(source, argKeys) {
+  var output = {};
+  var sourceKeys = Object.keys(source);
```

Comment:
Instead of converting `source` to an array just so you can use a `for` loop, you should leave it an object, and use a `for...in` loop